observationtree/src/main/scala/explore/observationtree/ObsBadge.scala

```diff
@@ -72,7 +72,7 @@ object ObsBadge {
                 <.div(
                   ExploreStyles.ObservationCardHeader,
                   props.layout match {
-                    case NameAndConf        => obs.name.orEmpty
+                    case NameAndConf        => obs.name.fold("")(_.value)
```

Comment:
Shall we use some placeholder here like `<UNNAMED>` or `-----` ?

Reply:
I recall a report of some people being annoyed at `<UNAMED>` 🤷

Reply:
Ah, let's avoid that one then (which we are not using anymore for new objects). How about dashes? The idea is to make explicit that there is no name. This situation shouldn't happen when using only the UI, but it can happen when using the API.

Reply:
Switched to dashes.